ql/src/test/results/clientpositive/llap/explainanalyze_acid_with_direct_insert.q.out

```diff
@@ -47,21 +47,21 @@ STAGE PLANS:
                 TableScan
                   alias: _dummy_table
                   Row Limit Per Split: 1
-                  Statistics: Num rows: 1/1 Data size: 10 Basic stats: COMPLETE Column stats: COMPLETE
+                  Statistics: Num rows: 1/1 Data size: ###Masked### Basic stats: COMPLETE Column stats: COMPLETE
                   Select Operator
                     expressions: array(const struct(1),const struct(2),const struct(3),const struct(4)) (type: array<struct<col1:int>>)
                     outputColumnNames: _col0
-                    Statistics: Num rows: 1/1 Data size: 72 Basic stats: COMPLETE Column stats: COMPLETE
+                    Statistics: Num rows: 1/1 Data size: ###Masked### Basic stats: COMPLETE Column stats: COMPLETE
```

Comment:
I don't think we should mask stats value, that could lead to hidden issues, better to update the sizes

Reply:
I too think its better to correct the data size. 
* We can infer quite a bit of info from the actual value and catch issues more easily(such as, if predicate push down is resulting in less data being read?)
* Commit history will also be preserved which will allow us to  track which commit modified the data size (Like the orc upgrade mentioned in previous comment by okumin)